daemon/src/projection.rs

```diff
@@ -1112,8 +1142,24 @@ impl Actor {
         );
     }
 
-    fn handle(&mut self, msg: Update<Option<model::MakerOffers>>) {
-        self.tx.send_offer_update(msg.0.into());
+    fn handle(&mut self, msg: Update<Vec<model::Offer>>) {
+        let new_offers = msg
+            .0
+            .into_iter()
+            .filter_map(|offer| match CfdOffer::new(offer, self.role) {
+                Ok(offer) => Some(offer),
+                Err(e) => {
+                    tracing::warn!("Failed to build CfdOffer from model::Offer: {e:#}");
+                    None
+                }
+            })
+            .collect_vec();
+
+        self.state.update_offers(new_offers);
```

Comment:
If we don't compose the state here we will only send the latest offers (these will reference a single currency pair at a time at the moment) to the feed.

I wanted to keep the pre-existing feed behaviour: on every update we send everything we know, even if it's redundant. It's not efficient (and I think we should not do this eventually), but I didn't want to introduce any unexpected changes.

Reply:
Right, because the maker does not send the offers alltogeher, but sends them by symbol - even thought the new protocol would allow sending them all together (but it's questionable if that is something we would want).
Ok, then let's keep the state for now and tackle this in follow up(s).